packit_service/worker/jobs.py

```diff
@@ -93,6 +93,11 @@ def get_handlers_for_event(
         ),
     ):
         handlers_triggered_by_comment = get_handlers_for_comment(event.comment)
+
+        if handlers_triggered_by_comment and not isinstance(
+            event, PullRequestCommentPagureEvent
```

Comment:
Careful about issue comment on Pagure, not sure if we handle those. Better safe than sorry…